website/custom-elements-manifest.config.js

```diff
@@ -15,6 +16,13 @@ export default {
     eventPlugin(),
     methodAndFieldPlugin('method'),
     methodAndFieldPlugin('field'),
-    sortMembersPlugin()
-  ]
+    sortMembersPlugin(),
+    expandTypesPlugin()
+  ],
+  
+  /** Overrides default module creation: */
+  overrideModuleCreation: ({ts, globs}) => {
+    const program = getTsProgram(ts, globs, "../tsconfig.all.json");
+    return program.getSourceFiles().filter(sf => globs.find(glob => sf.fileName.includes(glob.replace('../',''))));
```

Comment:
the files returned by `progragram.getSourceFiles()` contain the full path, not a path relative to this file. This path is then also put in the `custom-elements.json`. 
I can't find a way to remove that path, or make sure that it's relative.
@jpzwarte @anna-lach You want to give it a try?